test/fixtures/refract/x-summary-and-description.json

```diff
@@ -52,7 +52,9 @@
     {
       "element": "category",
       "meta": {
-        "classes": ["api"]
+        "classes": [
+          "api"
+        ]
```

Comment:
True, but I'm not sure why this is a change to this PR as it seems unrelated. Is it making things consistent? Or was this making tests fail?